lib/teleterm/daemon/daemon_test.go

```diff
@@ -0,0 +1,41 @@
+package daemon
+
+import (
+	"context"
+	"testing"
+
+	"github.com/gravitational/teleport/lib/teleterm/clusters"
+)
+
+type mockCluster struct {
+	cluster
```

Comment:
I would ditch this too - being used as the interface is enough to guarantee conformance.

Reply:
This is to demonstrate how you would override a single method of the cluster interface, for the purpose of testing, without having to stub out the rest of them.

Reply:
Gotcha. Hopefully whatever is being tested doesn't rely on all 10+ methods and could benefit from a smaller interface, but the technique is definitely useful. Embedding a concrete type might also make sense, depending on how it all works.